mlflow/store/artifact/ftp_artifact_repo.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, artifact_uri):
             "username": parsed.username,
             "password": parsed.password,
         }
-        self.path = parsed.path
+        self.path = parsed.path or "/"
```

Comment:
If the artifact looks like `ftp://admin:123@localhost:21` and doesn't contain the path, use `/` as the root.